index.html

```diff
@@ -0,0 +1,12 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css"/>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
Great job @wassimchakib 🤝

- [ ] As stated in the project requirements, kindly add a class to this element and then apply your styles to this class instead of applying the styles to the HTML element itself.